apps/sync/test/unit/crypto/stringconversion_test.js

```diff
@@ -17,76 +17,105 @@ requireApp('sync/js/crypto/main.js');
 requireApp('sync/test/unit/fixtures/fxsyncwebcrypto.js');
 
 suite('utils', () => {
-  suite('rawStringToByteArray', () => {
-    test('converts a raw string to a ByteArray', () => {
-     const ba = StringConversion.rawStringToByteArray('hi ✓');
```

Comment:
With what we know now, 'hi ' is not a good fixture here, please use `'\x00\xEA\x80\xFF'` instead, and then `expect(ba).to.deep.equal([0, 234, 128, 255]);` (or something similar, which contains bytes both above and below 128.